.github/workflows/daily_summary.yml

```
@@ -3,30 +3,30 @@ name: Daily HN Summary
 on:
   schedule:
     # Run at 8:00 UTC (adjust as needed)
-    - cron: '0 23 * * *'
-  
+    - cron: "0 23 * * *"
+
   # Allow manual triggering
   workflow_dispatch:
 
 jobs:
   summarize:
     runs-on: ubuntu-latest
-    
+
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
-      
+        uses: actions/checkout@v4
+
       - name: Set up Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@v5
         with:
-          python-version: '3.10'
-          cache: 'pip'
-      
+          python-version: "3.10"
```

```suggestion
          python-version: "3.13"
```

IMO: pythonも3.13までリリースされているので、bumpしても良いかも

FYI: https://www.python.org/downloads/release/python-3130/

---

そうですね！ありがとうございます！